src/main/java/apoc/export/csv/CsvFormat.java

```diff
@@ -75,10 +87,10 @@ private CSVWriter getCsvWriter(Writer writer, ExportConfig config)
         return out;
     }
 
-    public ProgressInfo dump(Result result, Writer writer, Reporter reporter, ExportConfig config) throws Exception {
+    public ProgressInfo dump(Result result, FileManagerFactory.ExportCypherFileManager writer, Reporter reporter, ExportConfig config) throws Exception {
         try (Transaction tx = db.beginTx()) {
```

Comment:
```suggestion
        try (Transaction tx = db.beginTx(); PrintWriter printWriter = writer.getPrintWriter("csv");) {
```
e togliere il `close`